.github/stale.yml

```diff
@@ -0,0 +1,12 @@
+daysUntilStale: 60
```

Comment:
Not sure why this is considered an addition as this file already exists in master and is identical?

Reply:
Not sure, I didn't edit that file.

Reply:
I checked your fork and your `submit` branch is 2 commits behind (the commit which added this file) so you'll need to rebase.